app/models/text_field_type.rb

```diff
@@ -25,14 +29,22 @@ def mapping_field_name
   end
 
   def text_present
-    errors.add(:text, "must be present") if @text.empty?
+    errors.add(:text, 'must be present') if @text.empty?
   end
 
   def text_length
     validator = LengthValidator.new(validations[:length].merge(attributes: [:text]))
     validator.validate_each(self, :text, text)
   end
 
+  def text_unique
+    errors.add(:text, 'Slug Must be unique') unless FieldItem.jsonb_contains(:data, text: text).empty?
```

Comment:
So, correct me if I'm wrong, but I think the `#jsonb_contains` method here will check all JSON fields across `FieldItem`s and check to see if any of them contain the string, not just `FieldItem`s for the same field. This could cause false errors to be raised if, for example, the title of a ContentItem is the same as it's SEO Title, even if the validation only exists on the slug field